bands/configspec.ini

```diff
@@ -13,18 +13,24 @@ default_performance_minutes = integer(default=40)
 
 
 [dates]
+# Add deadlines to enable certain checklist steps for certain group types
+# For example, band_panel_deadline enables the panel step for the band group type
```

Comment:
The changes to these config options allow the code to dynamically check for a deadline based on the model name.